syntax = "proto3";

service Store {
    rpc price (price_request) returns (price_response) {}
    rpc sale (sale_request) returns (sale_response) {}
    rpc end_of_work_store (end_of_work_store_request) returns (end_of_work_store_response) {}

}

message price_request {
}

message price_response {
    int32 price_value = 1; 
}

message sale_request {
    bytes transaction_key = 1; 
}

message sale_response {
    int32 status = 1; 
}

message end_of_work_store_request {
}

message end_of_work_store_response {
    int32 status = 1; 
}